cores/esp32/esp32-hal-touch-ng.c

```diff
@@ -129,7 +129,7 @@ bool touchStart() {
   if (running) {  // Already running
     return true;
   }
-  if (enabled && (touch_sensor_start_continuous_scanning(touch_sensor_handle) != ESP_OK)) {
+  if (!enabled && (touch_sensor_start_continuous_scanning(touch_sensor_handle) != ESP_OK)) {
```

Comment:
Hi, your approach will create a bug and touch will never start. PLease apply following fix!
```suggestion
  if(!enabled) {
    log_e("Touch sensor not enabled!");
    return false;
  }
  if (touch_sensor_start_continuous_scanning(touch_sensor_handle) != ESP_OK) {
```
+ change the error log message to:
`Touch sensor failed to start continuous scanning!`

Reply:
Oh, you're right!
I also saw the same logical issue in the `touchDisable` function and fixed it. Please take a look.